packages/devp2p/typedoc.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   extends: '../../config/typedoc.js',
   entryPoints: ['src'],
   out: 'docs',
-  exclude: ["test/**/*.ts", "examples/*.ts"],
+  exclude: ["test/**/*.ts", "examples/*.ts", "src/protocol/protocol.ts"],
```

Comment:
I think it would be okay to keep the generated documentation for the parent class, did you have a reason to exclude it?

Reply:
@holgerd77 Mentioned in one of his review comments that it might be a good idea to keep people from using it directly.